hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/crypto/HapiGetAccountBalance.java

```diff
@@ -87,6 +87,7 @@ public class HapiGetAccountBalance extends HapiQueryOp<HapiGetAccountBalance> {
     private boolean assertAccountIDIsNotAlias = false;
     private ByteString rawAlias;
 
+    private boolean includeTokenMemoOnError = false;
```

Comment:
I thought the "include token memo on error" was a different PR? (mentioned only here and not in all the files that include bits-and-pieces of the feature)

Reply:
Merged `develop`, this diff will be gone now.